boltdesignsystem/bolt
# WWW-467 page header design updates

## Jira

https://pegadigitalit.atlassian.net/browse/WWW-467

## Summary

Adds single level main site navigation item and misc. transition/animation to Page Header component.

## Details

1. Adds the ability to generate single level `ul` that doesn't create a full width dropdown. `flat` prop added to the ul template.
2. Adds `view_all` prop in the li template to handle "view all" links.
3. Removed certain styles from `full_width` prop and used them in `view_all` instead.
4. Link colors have been updated to address stakeholders' feedback.
5. Transitions of dropdowns have been enhanced to be smoother.
6. PegaWorld and Partners page header demos added.

## How to test

Run the branch locally and check the docs. Make sure the new props work as expected.
